docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -226,6 +227,86 @@ defs = Definitions(
 )
 ```
 
+### Checks on partitioned assets
+
+To write a check on a partitioned asset, summarize the status of all the partitions as a single pass/fail for the asset.
+
+Here's an example that checks that no partitions are null.
+
+```python file=/concepts/assets/asset_checks/partitions.py
+from typing import Dict
+
+from dagster import (
+    AssetCheckResult,
+    Definitions,
+    StaticPartitionsDefinition,
+    asset,
+    asset_check,
+)
+
+
+@asset(partitions_def=StaticPartitionsDefinition(["a", "b", "c"]))
+def partitioned_asset(context):
+    return "sample_data"
+
+
+@asset_check(asset=partitioned_asset)
+def no_nones(partitioned_asset: Dict[str, str]):
```

Comment:
Do we have a section on partitioned -> unpartitioned assets anywhere? Don't see it in https://docs.dagster.io/concepts/partitions-schedules-sensors/partitioning-assets, but this is explaining the same syntax

Reply:
Could you clarify what you mean by `partitioned -> unpartitioned assets` here? Like going from partitioned to not, or something else?

Reply:
Correct, an asset without partitions that depends on an asset with partitions

Reply:
Gotcha. No official docs, but could we use this to build off of? https://github.com/dagster-io/dagster/discussions/15375

Reply:
ideally we'd have that in docs and link to it from here, but I think this section is alright without it